routers/repo/issue.go

```diff
@@ -1542,6 +1549,11 @@ func ChangeCommentReaction(ctx *context.Context, form auth.ReactionForm) {
 
 	switch ctx.Params(":action") {
 	case "react":
+		if !util.IsStringInSlice(form.Content, setting.UI.Reactions) {
+			ctx.Flash.Error(ctx.Tr("settings.forbidden_reaction", form.Content))
```

Comment:
So after setting flash error you should then redirect back to the URL that sent the post